docs/source/reference.rst

```diff
@@ -235,3 +235,42 @@ Errors
    :members:
 .. autoclass:: dramatiq.results.ResultFailure
    :members:
+
+Environment Variables
+---------------------
+
+These are the environment variables that dramatiq reads
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Default
+     - Description
+   * - ``dramatiq_restart_delay``
+     - 3000 (3 seconds)
+     - The number of milliseconds to wait before restarting consumers after a connection error.
+   * - ``dramatiq_queue_prefetch``
+     - 2 times number of worker threads
+     - The number of messages to prefetch from the queue for each worker process. In-progress messages are included in the count.
+   * - ``dramatiq_delay_queue_prefetch``
+     - 1000 times number of worker threads
+     - The number of messages to prefetch from the delay queue for each worker.
+   * - ``dramatiq_dead_message_ttl``
+     - 604800000 (One week)
+     - The maximum amount of time a message can be in the dead queue for the RabbitMQ Broker (in milliseconds).
+   * - ``dramatiq_group_callback_barrier_ttl``
+     - 86400000 (One day)
+     - 
+   * - ``dramatiq_prom_lock``
+     - tempfile.gettempdir()/dramatiq-prometheus.lock
+     - The path to the file to use to race Exposition servers against one another.
```

Comment:
I don't know much about Prometheus but this env var is assigned to the `LOCK_PATH` global which is then never used.

I'd say let's elide it here and sort it out separately (from my investigation - it was unused since 1ff83f576f82380a4df57e9da9d333913a72f839, though there was a change to getenv in https://github.com/Bogdanp/dramatiq/pull/577).

Reply:
`LOCK_PATH` removal here https://github.com/Bogdanp/dramatiq/pull/722

Reply:
yep, have removed it from this PR